server/pbenchinacan/load_keycloak.sh

```diff
@@ -17,12 +17,53 @@
 # "http://localhost:8080/*" unless specified otherwise by 'KEYCLOAK_REDIRECT_URI'
 # env variable.
 
+KEYCLOAK_BASE_IMAGE=${KEYCLOAK_BASE_IMAGE:-"quay.io/keycloak/keycloak:latest"}
 KEYCLOAK_HOST_PORT=${KEYCLOAK_HOST_PORT:-"http://localhost:8090"}
+KEYCLOAK_HEALTH_URI="${KEYCLOAK_HOST_PORT}/health"
 KEYCLOAK_REDIRECT_URI=${KEYCLOAK_REDIRECT_URI:-"http://localhost:8080/*"}
 ADMIN_USERNAME=${ADMIN_USERNAME:-"admin"}
 ADMIN_PASSWORD=${ADMIN_PASSWORD:-"admin"}
 REALM=${KEYCLOAK_REALM:-"pbench"}
 CLIENT=${KEYCLOAK_CLIENT:-"pbench-server"}
+KEYCLOAK_IMAGE_TAG=${KEYCLOAK_IMAGE_TAG:-"latest"}
+
+container_name="mykeycloak"
+podman run -d --rm --name ${container_name} -p 8090:8090 \
+-e KEYCLOAK_ADMIN=admin -e KEYCLOAK_ADMIN_PASSWORD=admin \
+${KEYCLOAK_BASE_IMAGE} start-dev --health-enabled=true --http-port=8090
```

Comment:
Since you're using `-d` on the `podman` command, the invocation will print the container ID for you, so you don't need to give it an explicit name -- you can use the name which Podman generates.

This has the advantage that it will be unique, such that two concurrent invocations of this script won't interfere with each other.

E.g., I believe that you can just do the following:
```
container_name=$(podman run -d --rm --name ${container_name} -p 8090:8090 \
-e KEYCLOAK_ADMIN=admin -e KEYCLOAK_ADMIN_PASSWORD=admin \
${KEYCLOAK_BASE_IMAGE} start-dev --health-enabled=true --http-port=8090)
```

Reply:
Based on our discussion yesterday, I believe that this is now moot.